scanner/src/main/java/ca/on/oicr/gsi/runscanner/scanner/RestResponseController.java

```diff
@@ -41,10 +39,17 @@
    * Given a known run name. If no run is found, null is returned. If there are multiple runs with
    * the same name that are from different sequencers, one is randomly selected.
    */
-  @GetMapping(
-      value = "/run/{name}",
-      produces = {"application/json"})
-  public ResponseEntity<NotificationDto> getByName(@PathVariable("name") String id) {
+  @GetMapping("/run/{name}")
+  @ApiOperation(
+      value = "Get run by name",
+      response = ca.on.oicr.gsi.runscanner.dto.NotificationDto.class,
+      responseContainer = "ResponseEntity")
+  @ApiResponses({
+    @ApiResponse(code = 200, message = "dto"),
+    @ApiResponse(code = 404, message = "null")
+  }) // TODO: How do I use the HttpStatus naming instead of manually using the value?
```

Comment:
You'd need a constant int value, which means you can't use the HttpStatus or similar Status enums. You could create a class with the constants:

```
public class HttpStatusCodes {
  private static final int SUCCESS = 200;
  private static final int NOT_FOUND = 404;
  ...
}
```